README.md

```diff
@@ -184,6 +184,13 @@ On MacOS
 conda install pkg-config libuv
 ```
 
+On Windows
+```bash
+# Add these packages and set libuv_ROOT environment variable if torch.distributed is needed
+conda install -y -q -c rdonnelly libuv
+set libuv_ROOT={conda active env location}\Library
```

Comment:
I wonder why MacOS does not require this? MacOS requires install pkg-config instead. Can we do the same for Windows?

Reply:
In Gloo PR https://github.com/pytorch/pytorch/pull/45392, I first tried with pkg-config, but Windows doesn't come with pkg-config, later on, base on the code review suggestion, I switched to use find_library and find_file to work with libuv_ROOT. So libuv_ROOT need to be set as a environment variable, or defined in cmake command line as -Dlibuv_ROOT.

Reply:
> but Windows doesn't come with pkg-config, later on, base on the code review suggestion

Does this mean even if we do `conda install pkg-config libuv`, we still cannot find libuv on Windows automatically?